sdf/final-project/presentation.md

```diff
@@ -10,12 +15,15 @@ This is intended to be filled out by staff members of DPI Tech Apprenticeship pr
 - [ ] **Introduction**: Does the presentation start with an introduction covering previous work/life experiences and connection to the problem?
 - [ ] **Target User**: Does the presentation clearly identify the target user?
 - [ ] **Target Problem**: Does the presentation clearly identify the problem the project addresses?
-- [ ] **Market Research and Problem Validation**: Is there evidence of market research or validation efforts, such as discussions with potential users, references to previous work/life experiences, and quanitative analysis?
+<!-- (at least 2/3) -->
+- [ ] **Market Research and Problem Validation**: Is there evidence of market research or validation efforts, such as discussions with potential users, references to previous work/life experiences, and quanitative analysis? 
 - [ ] **Solution Appeal and Usability**: Is the proposed solution presented in an engaging and compelling manner, making it easy to envision real users benefiting from and engaging with the application?
-- [ ] **Professional Preparation**: Is there a clear connection between the project work and the skills or experiences that prepare trainees for professional roles, potentially linking to industry categories or needs (e.g., as outlined on https://rfp.dpi.dev)?
+<!-- maybe make professional preparation it's own section -->
+- [ ] **Professional Preparation**: Is there an explicit connection between the project work and the skills or experiences that prepare trainees for professional roles, potentially linking to industry categories or needs (e.g., as outlined on https://rfp.dpi.dev), or connecting to a real business?
 - [ ] **Compelling**: Is there a hook? Is the presentation compelling/captivating?
 - [ ] **Transition and Flow**: Are transitions between slides smooth and logical, contributing to a cohesive flow of the presentation without causing distractions?
-- [ ] **Closing**: Does the presentation have a strong closing? <!-- what does this mean? -->
+<!-- what does this mean? do we mean "clear" closing? Or require that the closing ties back to the problem or something discussed earlier (key takeaways, quote, etc.) -->
```

Comment:
I think it's good to have a closing that ties back to the problem and a call to action as opposed to just a "Thank you"

Reply:
Agreed I think we should stick to a "strong" closing as apposed to a clear closing. I feel like this would avoid pretty much everyone getting the point for just saying "Thank You". Tying the project back to the problem and how it helps solve it then it makes it strong.

Reply:
We could potentially separate this into a point for a "clear" closing vs. a "strong" closing, since that seemed to be a minor sticking point. Some people had much *stronger* closings compared to those who just had *clear* closings - should they be getting the same amount of points?